db/schema.rb

```diff
@@ -124,6 +124,8 @@
     t.boolean "dev_marketing_emails", default: false
     t.string "category", default: "participant"
     t.string "country"
+    t.boolean "quality_acceptance", default: false
+    t.boolean "disqualify_acceptance", default: false
```

Comment:
Looks like this needs to be rebased on latest master?

Reply:
Rebased before pushing  `586a0fc`, so should be good there.

Reply:
Branch still shows up as many commits behind master, might need a `git fetch origin && git rebase origin/master`?

Reply:
Just pushed up a rebase with the recent copy changes